pylint/checkers/variables.py

```diff
@@ -2430,6 +2423,17 @@ def _check_unpacking(self, inferred, node, targets):
                 args=(_get_unpacking_extra_info(node, inferred),),
             )
 
+    @staticmethod
+    def _get_values_to_unpack(node: nodes.Assign) -> Optional[List]:
+        """Return the list of values of the `Assign` node"""
+        if isinstance(node, (nodes.Tuple, nodes.List)):
+            return node.itered()
+        if isinstance(node, astroid.Instance) and any(
+            ancestor.qname() == "typing.NamedTuple" for ancestor in node.ancestors()
+        ):
+            return [i for i in node.values() if isinstance(i, nodes.AssignName)]
```

Comment:
Shouldn't this also cover `AnnAssign`?

Reply:
Never mind, I see this is just copied from previous code. Perhaps a follow-up issue worthy?

Reply:
Is that situation possible without running into E0001?
For example:
`a, b: List = [1,2]` # only single target (not tuple) can be annotated)?

Reply:
Hmm, seems I wasn't the sharpest when I reviewed this previously. You're right 😄

Reply:
I know how you feel!